### ISSUE TITLE
Auto TCAS System

### ISSUE THREAD

**Issue by derl30n** [flybywiresim/a32nx#2719]
**Is your feature request related to a problem? Please describe.**

**Describe the feature you'd like to see implemented**

I believe this might be a neat feature for the future.
Auto TCAS basically follows the instructions of the TCAS to automatically solve the situation.

Besides the logic and functionality, the FMA would need TCAS blue and TCAS active modes. (I am sure in the future the FMA "handling" is done differently.

**References**

https://www.youtube.com/watch?v=H35wuyqIRjE

**Additional context**

Discord username (if different from GitHub): DerL30N#3751

**Comment by St54Kevin** [comment]
I think this HAS to be an option in the EFB. With all the online traffic and all the weird pilots out there, I certainly DONT want the AP to follow it in this case :D

**Comment by beheh** [comment]
You could switch TCAS from TA/RA to RA, no?

**Comment by nikitanyc** [comment]
Hey, I came across a TCAS while I was messing with the ND-radar system. I tried to implement the "OFF" switch - since it only goes WX, WX/TURB, TURB, and MAP. 

Apologies for the horrible video quality, but is this something that someone was working on and left? Or is this an MSFS yet-unreleased system? It works during multiplayer and picks up AI traffic as well. It even shows the position of my aircraft. I can't switch out of the screen though.

https://www.youtube.com/watch?v=r6a9Jdxy27E%5D&feature=youtu.be
